sympy/concrete/expr_with_intlimits.py

```diff
@@ -23,7 +26,7 @@ def change_index(self, var, trafo, newvar=None):
         Usage
         =====
 
-        ``change_index(expr, var, trafo, newvar=None)`` where ``var`` specifies the
+        ``change_index(expr, var, trafo, newvar = None)`` where ``var`` specifies the
```

Comment:
PEP8 mandates blanks around `=` for assignments but not for keyword arguments and default parameters.